lib/command/src/Obelisk/Command/Project.hs

```diff
@@ -307,3 +308,26 @@ projectShell root isPure shellName command = do
     & nixShellConfig_common . nixCmdConfig_target . target_path ?~ "default.nix"
     & nixShellConfig_common . nixCmdConfig_target . target_attr ?~ ("shells." <> shellName)
   void $ liftIO $ waitForProcess ph
+
+findProjectAssets :: MonadObelisk m => FilePath -> m Text
+findProjectAssets root = do
+  let importableRoot = toNixPath root
+  isDerivation <- readProcessAndLogStderr Debug $
+    proc nixExePath
+      [ "eval"
+      , "-f"
+      , root
```

Comment:
Might as well `setCwd` (in the Cli module) and just use constant `"."` here.

Reply:
Actually, is there a reason to use `-f` *and* `import ...`?

Reply:
no, I think it's a no-op actually.

Reply:
Note that I applied this improvement already to other places in #653